### ISSUE TITLE
'current_user' endpoint returns Error 405 (stopped working on 24th of July)

### ISSUE THREAD

**Issue by JulienMaille** [ping/instagram_private_api#291]
This issue is very similar to https://github.com/ping/instagram_private_api/issues/266 except it happens when using `current_user()` from `instagram_private_api`

The error returned is `Error parsing error response: Expecting value: line 1 column 1 (char 0)` but the real problem is a `HTTP Error 405: Method Not Allowed` when calling https://github.com/ping/instagram_private_api/blob/526f244bae18962b5777e1f2f870699c0e4a8d2d/instagram_private_api/endpoints/accounts.py#L76-L82

I have this problem with 2 different accounts and other endpoints do work fine so I believe the problem is not on my end.
It has been working fine until the 23/24 of July when it suddenly started returning a 405

**Comment by JulienMaille** [comment]
closed with 37074e3a0f1369c813151ac5982d4d357599c4bd

**Comment by olegklimakov** [comment]
Still not working
urllib.error.HTTPError: HTTP Error 405: Method Not Allowed
```social-worker-ZY-9Oqoh/lib/python3.8/site-packages/instagram_private_api/client.py", line 523, in _call_api
    response = self.opener.open(req, timeout=self.timeout)```
